File: Java/Day One/SimpleArithmetic.java
import java.util.Random;
import java.util.Scanner;

public class SimpleArithmetic{

	public static void main (String... args){
		Scanner scanner = new Scanner(System.in);

		System.out.println(showMenu());
		Long timeOne = System.currentTimeMillis();
		int passes = 0;
		int numberPlay = 10;
		while (numberPlay > 0){
			int count = 0;
			int[] game = generateAndSubtractRandomNumbers();
			while(count < 2){
				System.out.print("What is " + game[0] + " - " + game[1] + ": ");
				int choice = scanner.nextInt();
				if (choice == game[2]){
					System.out.println("Bravo! That's great");
					passes += 1;
					numberPlay -= 1;
					break;
				}
				else{
					if (count == 1){
					System.out.println("You have exhasted your last trial");
						break;
						
					}
					System.out.println("You can do it sweetheart, Try Again");
					numberPlay -= 1;
					
			
			count += 1;
			}

		}


		}
		Long timeTwo = System.currentTimeMillis();
		Long timeUsed = timeTwo - timeOne;
		Long timeInSeconds = timeUsed/1000;
		System.out.println("The time used is: " + timeInSeconds + "Seconds");
		System.out.print("Your Total score is: "+ passes +"/10");


	}

	public static int[] generateAndSubtractRandomNumbers(){
		Random random = new Random();

	int numberOne = random.nextInt(20,51);
	int numberTwo = random.nextInt(1, 20);
	int subtract = numberOne - numberTwo;
	int[] result = {numberOne, numberTwo, subtract};

	return result;


	}

	public static String showMenu(){

	String menu ="""
*********************************************************
*	Welcome Victorious Jewel!!!			*
*	Let  Solve Subtraction Arithmetic Problem	*
*	Your Writing Material With You			*
*	Get Ready For An Amazing Experince		*
*	Bring Out The Genius in You			*
*********************************************************
""";
	return menu;
	
	}






}

File: JavaScript/Day One/SimpleArithmetic.js
const prompt = require('prompt-sync')();
const dateTime = new Date();


function generateAndSubtractRandomNumbers(){
	let numberOne = Math.floor(Math.random() * 31) + 20
	let numberTwo = Math.floor(Math.random() * 21) 
	let subtract = numberOne - numberTwo;
	let result = [numberOne, numberTwo, subtract];

	return result;


	}
function showMenu(){

	let menu =`
*********************************************************
*	Welcome Victorious Jewel!!!			*
*	Let  Solve Subtraction Arithmetic Problem	*
*	Your Writing Material With You			*
*	Get Ready For An Amazing Experince		*
*	Bring Out The Genius in You			*
*********************************************************
`;
	return menu;
	
	}		

const secondsBegins = dateTime.getSeconds()
console.log(showMenu());
let passes = 0;
let numberPlay = 10;
while (numberPlay > 0){
	let count = 0;
	let game = generateAndSubtractRandomNumbers();
		while(count < 2){
			choice = prompt("What is " + game[0] + " - " + game[1] + ": ");
				if (Number(choice) == game[2]){
					console.log("Bravo! That's great");
					passes += 1;
					numberPlay -= 1;
					break;
				}
				else{
					if (count == 1){
					console.log("You have exhasted your last trial");
						break;
						
					}
					console.log("You can do it sweetheart, Try Again");
					numberPlay -= 1;
					
			
			count += 1;
			}

		}


}

const endindDateTime = new Date();
const secondsEnds = endindDateTime.getSeconds()

let secondsUsed = secondsEnds - secondsBegins

console.log("Your Total score is: "+ passes +"/10");
console.log("Your Total time spent on answering the question  is: "+ secondsUsed +"seconds");




